src/_press/four-northern-california-transit-agencies-join-forces-to-buy-contactless-open-loop-fare-payment-systems-off-of-californias-purchasing-agreements.md

```diff
@@ -0,0 +1,36 @@
+---
+date: "2023-08-18T17:00:00-07:00"
+title: Four Northern California transit agencies join forces to buy contactless open-loop fare payment systems off of California’s purchasing agreements
+heading: Four Northern California transit agencies join forces to buy contactless open-loop fare payment systems off of California’s purchasing agreements
+lead: |-
+  Humboldt Transit Authority, Lake Transit Authority, Mendocino Transit Authority, and Redwood Coast Transit Authority install a modern fare system that accepts riders’ bank cards and mobile wallets
+intro: |-
+  Transit riders in four counties north of the San Francisco Bay Area can now tap to ride the bus on four public transit agencies—and seamlessly transfer between the agencies’ dial-a-ride vehicles or local and regional lines at shared bus stops—without stopping to download any apps, purchase or reload multiple agency farecards, or juggle exact change.
+tags:
+  - Contactless Payments
+outlet: "Cal-ITP"
+---
+
+Humboldt Transit Authority, Lake Transit Authority, Mendocino Transit Authority, and Redwood Coast Transit Authority—collectively part of the Far North Group—are the first group of agencies to band together to use the State of California’s competitively priced contracts to purchase and install the hardware and software services needed for accepting customers’ debit and credit cards and mobile wallets for fare payments.
+
+Cal-ITP—Caltrans’ California Integrated Travel Project—supported the agencies through the process of purchasing open-loop payment acceptance devices and fare calculation software from the State of California’s competitively awarded [Master Service Agreements](https://www.calitp.org/assets/Contactless.Payments.MSA.pdf?utm_source=Press+Release&utm_medium=Website&utm_id=FNG) (MSAs), contracts that allow U.S. transit agencies to purchase directly from vendors without further competitive bidding.
```

Comment:
Most of the links in the asset document contained this stuff at the end: `?utm_source=Press+Release&utm_medium=Website&utm_id=FNG`. Is that actually desired, or should I remove those?

Reply:
@esquared415 @mrose914 - See question above

Reply:
Got it! Thanks